ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -0,0 +1,734 @@
+            DC1_L2_LEAVES:
+        CONNECTED_ENDPOINTS:
+          children:
+            DC1_L3_LEAVES:
+            DC1_L2_LEAVES:
+```
+
+The `NETWORK_SERVICES` section does two things:
+
+1. It references the `ansible-avd-examples/single-dc-l3ls/NETWORK_SERVICES.yml` group variable file that specifies the VRFs and VLANs used in the entire fabric.
+2. Defines the scope of the fabric where these VRFs and VLANs are applied, in this case the two groups `DC1_L3_LEAVES` and `DC1_L2_LEAVES`
+
+The `CONNECTED_ENDPOINTS` section also does two things:
+
+1. It references the `ansible-avd-examples/single-dc-l3ls/CONNECTED_ENDPOINTS.yml` group variable file that specifies all endpoints, which would typically be servers.
+2. Defines the scope of where these endpoints are connected to the fabric, in this case the two groups `DC1_L3_LEAVES` and `DC1_L2_LEAVES`.
+
+## Defining device types
+
+Since we're building a L3LS network, we must tell Ansible AVD which devices are spines, L3 leaves, L2 leaves etc. We've already grouped the devices in the inventory and so in this example this is done using the group variable files with the following names and content:
+
+`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_SPINES.yml`
+
+```yml
+---
+type: spine
+```
+
+`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_L3_LEAVES.yml`
+
+```yml
+---
+type: l3leaf
+```
+
+`ansible-avd-examples/single-dc-l3ls/group_vars/DC1_L2_LEAVES.yml`
+
+```yml
+---
+type: l2leaf
+```
+
+For example, this means that all switches that are children of the DC1_SPINES group defined in the inventory, will be of the type `spine`.
+
+## Setting fabric-wide configuration parameters
+
+In the `ansible-avd-examples/single-dc-l3ls/group_vars/FABRIC.yml` file, we define generic settings that apply to all children of the `FABRIC` group as defined in the inventory described earlier.
+
+The first section defines how the Ansible host connects to the devices. In this case we're specifying that:
+
+- The Ansible host must use eAPI via HTTPS.
+- Network OS which in this case is Arista EOS.
+- The username/password combo.
+- How to escalate privileges to get write access.
+- We're using SSL.
+- We're not validating certs.
+
+```yml
+ansible_connection: ansible.netcommon.httpapi
+ansible_network_os: arista.eos.eos
+ansible_user: ansible
+ansible_password: ansible
+ansible_become: true
+ansible_become_method: enable
+ansible_httpapi_use_ssl: true
+ansible_httpapi_validate_certs: false
+```
+
+In the next section we specify variables that generate configuration to be applied to all devices in the fabric:
+
+- The name of the fabric for internal Ansible AVD use.
+- Local users/passwords and their privilege levels. In this case the "ansible" user is set with the password "ansible".
+- BGP peer groups and their passwords (all passwords are "arista").
+- Point-to-point interface MTU, in this case it's set to 1500 since the example is using vEOS, but if using hardware this should be set to 9214 instead.
```

Comment:
```suggestion
- Point-to-point interface MTU, in this case it's set to 1500 since the example is using vEOS, but when using hardware this should be set to 9214 instead.
```